init.sql

```diff
@@ -152,502 +152,84 @@ INSERT INTO zip_codes (id, sid, default_state, city, default_type, zip_code, gro
+INSERT INTO trees(ID,growing_zone,tree_name,botanical_name,descrip1,descrip2,descrip3,descrip4,descrip5,descrip6,grows_in_zones,sun_preference,soil_preference,mature_height,mature_spread,growth_speed,flower_color,foliage_color,fall_color,pollinator_friendly,outgoing_link,main_tree_image,tree_image_1,tree_image_2,tree_image_3,tree_image_4) VALUES (54,9,'Autumn Brilliance Serviceberry','Amelanchier x grandiflora ''Autumn Brilliance''','Native Tree with Four Seasons of Interest','Superb Size for Smaller Lots','Pink Buds Open to Masses of Lacy, White Flowers','Butterflies and Beneficial Pollinators Adore the Delicate Blooms','Wonderful Food Source for Birds','Native Disease-Resistance','3-9','Full Sun, Partial Shade','Well Drained','20 - 25 feet','15 - 20 feet','Medium','White','Blue-Green','Orange, Red','Yes','https://www.naturehills.com/autumn-brilliance-serviceberry','https://raw.githubusercontent.com/irmakerdem/tree-pal-api/main/assets/autumn-brilliance/autumn-brilliance-serviceberry-multi-stemmed.webp','https://raw.githubusercontent.com/irmakerdem/tree-pal-api/main/assets/autumn-brilliance/autumn-brilliance-serviceberry-blooms.webp','https://raw.githubusercontent.com/irmakerdem/tree-pal-api/main/assets/autumn-brilliance/autumn-brilliance-serviceberry-full-600x600.webp','https://raw.githubusercontent.com/irmakerdem/tree-pal-api/main/assets/autumn-brilliance/autumn-brilliance-serviceberry-4-800x800.webp','https://raw.githubusercontent.com/irmakerdem/tree-pal-api/main/assets/autumn-brilliance/autumn-brilliance-serviceberry-fall-color.webp');
```

Comment:
Awesome job reseeding the database! Trees sometimes struggle to grow without proper seeding 😁